src/main/scala/tech/sourced/gemini/HashSparkApp.scala

```diff
@@ -55,29 +66,15 @@ object HashSparkApp extends App with Logging {
         LogManager.getRootLogger().setLevel(Level.INFO)
       }
 
-      val repos = listRepositories(reposPath, spark.sparkContext.hadoopConfiguration, config.limit)
-      println(s"Hashing ${repos.length} repositories in: $reposPath\n\t" + (repos mkString "\n\t"))
-
       val gemini = Gemini(spark, log, "hashes")
       CassandraConnector(spark.sparkContext).withSessionDo { cassandra =>
         gemini.applySchema(cassandra)
       }
-      val filesToWrite = gemini.hash(reposPath, config.limit)
+      val filesToWrite = gemini.hash(reposPath, config.limit, config.format)
       gemini.save(filesToWrite)
       println("Done")
 
     case None =>
       System.exit(2)
   }
-
-  private def listRepositories(path: String, conf: Configuration, limit: Int): Array[Path] = {
```

Comment:
1. if this is removed - it would make sense to cleanup imports as well.
2. we want to keep this, as it's very useful for user feedback

It already uses HDFS FS API that supports local and remote FS, but needs to be improved though to support different formats i.e bucketed .siva files in HDFS.